inst/include/Rcpp/exceptions.h

```diff
@@ -27,21 +27,22 @@
 #define RCPP_DEFAULT_INCLUDE_CALL true
 #endif
 
-#define GET_STACKTRACE() stack_trace( __FILE__, __LINE__ )
```

Comment:
Although unlikely it's possible that some client package of Rcpp makes use of this macro, so this could fail in revdep checks.